src/postgraphile/cli.ts

```diff
@@ -425,6 +426,16 @@ const {
   jwtVerifyIgnoreNotBefore,
   jwtVerifyIssuer,
   jwtVerifySubject,
+  jwtSignAlgorithm,
+  jwtSignKeyId,
+  jwtSignExpiresIn,
+  jwtSignNotBefore,
+  jwtSignAudience,
+  jwtSignSubject,
+  jwtSignIssuer,
+  jwtSignJwtId,
+  jwtSignNoTimestamp,
+  jwtSignEncoding,
```

Comment:
We currently allow passing through `jwtVerifyOptions` in library mode (see [library usage](https://www.graphile.org/postgraphile/usage-library/)); instead of accepting all these options can we just accept one called `jwtSignOptions` instead?

Reply:
Should we deprecate the use of `jwtVerify***` options then, to provide the equal configuration possibilities for both signing and verifying? It seems to be surprising for users to have `jwtVerify*` variables and have no `jwtSign*` ones. What do you think?

Reply:
Yeah; I'll probably remove them in v5 so it makes sense to deprecate them. I think we'll probably accept a JSON5 object on the CLI in v5; so we can just do `--jwt-sign '{issuer:"postgraphile",noTimestamp:true}'`

Reply:
(Give me a shout when you're ready for me to re-review.)